modules/EvseManager/EvseManager.cpp

```diff
@@ -1238,19 +1247,30 @@ void EvseManager::log_v2g_message(Object m) {
 
     std::string xml = "";
     std::string json_str = "";
-    if (m["v2g_message_xml"].is_null() and m["v2g_message_json"].is_string()) {
+    std::string exi_hex = "";
+    std::string exi_base64 = "";
+
+    if (not m["v2g_message_xml"].is_null()) {
```

Comment:
Not related to this PR, but what is `Object`? in VSCode it doesn't give me the info.
I can't be sure that `std::string msg = m["v2g_message_id"];` is safe (a few lines up) given that `m["v2g_message_xml"].is_null()` is used.

Reply:
`Object` is a `nlohmann::json` alias for a `std::map`. So `const std::string xml = m.value("v2g_message_xml", "");` does not work here. But this is definitely mixed up. I'll change that.